portfolio/src/main/java/com/google/sps/data/AUX.java

```diff
@@ -60,9 +96,21 @@ public static Video VideoIdToObject(String videoId) throws IOException, GeneralS
         YouTube.Search.List searchRequest = youtubeService.search()
             .list("snippet")
             .setType("video");
-        SearchListResponse searchResponse = searchRequest.setQ(query)
-            .setKey(DEVELOPER_KEY)
-            .execute();
+
+        SearchListResponse searchResponse = new SearchListResponse();
+        try{
+            searchResponse = searchRequest.setQ(query)
+                .setKey(DEVELOPER_KEY)
+                .execute();
+        }
+        catch(Exception e){
+            //Print errors to console
```

Comment:
Please set a boolean DEBUG flag at the top and use it to gate the print statement.